### PR TITLE
DDCE-2642 : upgrade libs

### PR DESCRIPTION
**Pull request by lmugnai**

### CODE REVIEW

File: project/AppDependencies.scala [hunk 1]
```diff
@@ -2,7 +2,7 @@ import play.sbt.PlayImport._
 import sbt._
 
 object AppDependencies {
-  val bootStrapPlayVersion = "6.2.0"
+  val bootStrapPlayVersion = "7.0.0"
```
Comment (frikit) [inline project/AppDependencies.scala:5]: align version here as in the other project to 7.1

**Review by frikit — CHANGES_REQUESTED** [review]
- Please fix ${logger.uk.gov:-WARN} in application-json-logger.xml to be WARN
- Please fix ${logger.application:-WARN} in application-json-logger.xml to be WARN
- Please remove play.crypto.secret from application.conf
- Please add scalafmt.conf
- Please add run_all_tests.sh script
- Please add scalastyle.xml config
- Please update scalatest to 3.2.13

File: project/AppDependencies.scala (hunk 1 above)
Comment (lmugnai) [reply project/AppDependencies.scala]: done

**Comment by lmugnai** [comment]
Please fix ${logger.uk.gov:-WARN} in application-json-logger.xml to be WARN - **Done**
Please fix ${logger.application:-WARN} in application-json-logger.xml to be WARN - **Done**
Please remove play.crypto.secret from application.conf - **Done**
Please add scalafmt.conf - **Won't do. Discussed and this seems overkill for this repo**
Please add run_all_tests.sh script - **Won't do. Don't think there is a need for a script if it's simply 'sbt test'**
Please add scalastyle.xml config - **Won't do. Discussed and this seems overkill for this repo**
Please update scalatest to 3.2.13 - **Done**

**Review by frikit — APPROVED** [review]

**Pull request merged**